analysis_options.yaml

```diff
@@ -1,2 +1 @@
 analyzer:
```

Comment:
Just delete this file entirely if it's not configuring anything.

Reply:
done